templates/long.yaml

```
@@ -16,3 +16,11 @@
 
 - name: Test Project - Empty
   type: Project
+
+- name: Test Team
+  type: Team
+  can_public_join: false # true for teams which members can join without an invitation or approval
+  description: Test team description
+  members:
+  - principal_id: 11111
+  - email: "test_email@test.com"
```

I updated the list of members to use a [real test account](https://www.synapse.org/#!Profile:3426116/profile). In this case, the principal ID and email both point to the same, but that should be fine for the purposes of demonstration and testing. 

[Nit:](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.) I also removed the command for `can_public_join` because I think that information should live in the specification that I will be drafting soon. Otherwise, we would have to backfill comments for other fields, but I don't think this minimal template is the place for that. 

```suggestion
- name: Test Team
  type: Team
  can_public_join: false
  description: Test team description
  members:
  - principal_id: 3426116
  - email: synapseformation-test-user@sagebase.org
```